libs/ssl-util/src/Ssl/Util.hs

```diff
@@ -33,16 +31,17 @@ module Ssl.Util
   )
 where
 
-import Control.Exception
+import Control.Exception hiding (catch)
 import Data.ByteString.Builder
 import Data.Byteable (constEqBytes)
 import Data.Dynamic (fromDynamic)
 import Data.Time.Clock (getCurrentTime)
 import Imports
 import Network.HTTP.Client.Internal
 import OpenSSL.BN (integerToMPI)
-import OpenSSL.EVP.Digest (Digest, digestLBS)
-import OpenSSL.EVP.PKey (SomePublicKey, toPublicKey)
+import OpenSSL.EVP.Digest (digestLBS)
+import OpenSSL.EVP.Internal
+import OpenSSL.EVP.PKey
```

Comment:
I think this should be reverted given there was no actual code change in this file, looks like we depend on some Internal module for no reason.